redash/query_runner/qubole.py

```diff
@@ -24,6 +25,11 @@ def configuration_schema(cls):
         return {
             "type": "object",
             "properties": {
+                "query_type": {
+                    "type": "string",
+                    "title": "Query Type (quantum / presto / hive)",
+                    "default": "quantum"
```

Comment:
```suggestion
                    "default": "hive"
```

To prevent existing deployments breaking we need to keep this the same as it was in previous versions :-(

If you think it makes sense, you can create another query runner (just a subclass of this one) which is only for Quantum and will use this setting hard coded.

Reply:
Done. Made hive as default query type.